### PR TITLE
chore(common): upgrade dependencies

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR upgrades some dependencies and match them to have the same version across the repository.

### CODE REVIEW

File: packages/lib-helpers/react/package.json [hunk 1]
```diff
@@ -11,6 +11,6 @@
   "devDependencies": {
     "@types/react": "^18.2.56",
     "@types/react-dom": "^18.2.19",
-    "typescript": "^5.3.3"
+    "typescript": "5.3.3"
```
Comment (demetriusfeijoo) [inline packages/lib-helpers/react/package.json:14]: Hey @eunjae-lee 🙌 
Should the same change also be applied to the other packages using typescript? 🤔 

I mean, there are other projects using TS but not enforcing it to be the exact version as here.

**Review by Dawntraoz — APPROVED** [review]
LGTM! 🎉

Just a question, is there any changes in Vite we need to be aware, or they didn't change anything in those versions that we need to pay attention to?

File: packages/lib-helpers/react/package.json (hunk 1 above)
Comment (eunjae-lee) [reply packages/lib-helpers/react/package.json:14]: Thanks! Applied https://github.com/storyblok/field-plugin/pull/411/commits/5dc97986f28407a5f01b04e3a7ba642dd8abf7c9

**Comment by eunjae-lee** [comment]
> LGTM! 🎉
> 
> Just a question, is there any changes in Vite we need to be aware, or they didn't change anything in those versions that we need to pay attention to?

Nothing! It was just a minor upgrade and nothing changed in terms of usage, AFAIK :D

**Pull request merged**